README.rst

```diff
@@ -36,6 +36,10 @@ The dev environ provides:
 Install
 -------
 
+
+Install from source
```

Comment:
Can we use subsection headings here so they all fall under the `Install` main section?

```
Install from source
++++++++++++++
```
==============
```

Reply:
yes thats better